### ISSUE TITLE
The calculation problem of loss function in code

### ISSUE THREAD

**Issue by yuChen-XD** [HustMinsLab/GEAM#1]
Hello, it is an honor to read your paper on community detection titled 'A graph enhanced attention model for community detection in multiple networks'. Thank you very much for providing clear code. However, there is a question I would like to ask:

1. In the code, I couldn't find the corresponding calculations for formulas (20) to (24) mentioned in section 3.3 of the paper. It seems that the loss function in the code only uses formula (11). Can you answer my confusion?
Can you explain the corresponding meaning of 'loss_kl' in lines 99-108 of the train.py file?

2. In the code, you provided the AUCS dataset. Can you provide other datasets for complete testing and research?

I am conducting research on community detection and we will ensure that your article is cited in our future work. Thank you!
